src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -193,14 +192,15 @@ private enum Mode
             foreach (var url in urls)
             {
                 attributes["src"] = HtmlEncoder.HtmlEncode(url);
-                var content = string.Equals(url, staticSrc, StringComparison.OrdinalIgnoreCase)
-                    ? originalContent
-                    : string.Empty;
+                var content =
+                    string.Equals(url, staticSrc, StringComparison.OrdinalIgnoreCase)
+                        ? originalContent
+                        : null;
```

Comment:
nit: ugly and why change anything here at all?  put everything on one line if possible or at least restore previous wrapping.